lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
```

Comment:
Ah ok, there is currently an aria-label on the child <button> element. Should I move that aria-label to this input instead?